docker-compose.yml

```diff
@@ -2,9 +2,19 @@ version: '3.9' # optional since v1.27.0
 services:
   kabooty:
     build: .
-    image: kabooty/39 ## !!TODO: build version, triggers a deployment
+    image: kabooty/82 ## !! TODO: build version, triggers a deployment
     ports:
       - '3000:3000'
+    restart: unless-stopped
+    deploy:
+      mode: replicated
+      replicas: 1 ## !! TODO: implement replicas for extra redundancy
+    healthcheck:
+      test: ['CMD', 'curl', '-f', 'http://localhost:3000/api/healthy']
+      interval: 5s
+      timeout: 30s
+      retries: 3
+      start_period: 10s
```

Comment:
this might look like i changed it this is just coppied from master.

This branch was a few versions behind mb